content/docs/guide.md

```diff
@@ -1,94 +1,301 @@
+You can track the progress in the [related issue](https://github.com/packit/packit-service/issues/556) on GitHub.
+
+
+## 2. Approval
+
+As a next step, you need to have a valid [Fedora Account System](https://fedoraproject.org/wiki/Account_System)
+account to be able to start using Packit Service in an upstream project.
+For GitHub, we have an automated way of self-approval,
+for other instances, [contact us](/#contact) and we will approve you manually.
+We require our new users to have `GitHub Username` field set in the FAS account so that we can automatically
+check for the match between the `GitHub Username` field in the FAS account and the GitHub account that triggers the verification.
+(If you don't know, where to set this value,
+the URL looks like this: `https://accounts.fedoraproject.org/user/{my-fas-username}/settings/profile/#github`.)
+
+During the installation, we try to check the FAS account with the same
+username as the GitHub account that triggers the installation and if the check
+fails, we create a new issue in our
+[allowlist tracker](https://github.com/packit/notifications/issues/).
+(You will be tagged in that issue and should get a GitHub notification about that.)
+
+![Packit approval issue](/images/guide/guide_approval_issue.png)
+
+You can provide the FAS account in a comment
+as `/packit verify-fas <my-fas-username>` in the issue we create
+for your installation. This will trigger the automatic check.
+Until that, you will get a neutral status with `Namespace is not allowed!`
+message on your commits.
+
+![Packit approval issue: add mapping in the FAS settings](/images/guide/guide_approval_fas_settings.png)
+![Packit approval issue: add mapping by a comment](/images/guide/guide_approval_comments.png)
+
+Besides that:
+* If you want to configure builds via Packit,
+  your software needs to comply with [Copr guidelines](https://docs.pagure.org/copr.copr/user_documentation.html#what-i-can-build-in-copr)
+  since we use [Fedora Copr](src.fedoraproject.org/) for the builds.
+  Therefore, please, make sure you read them before configuring the [Packit build job](/docs/configuration/#copr_build).
+* If you are interested in using [internal instance of the Testing Farm](https://docs.testing-farm.io/general/0.1/services.html#_red_hat_ranch)
+  to run your tests in, please, [reach out to us](/#contact), since for this job, an additional approval on our side is needed.
+* For retrying the [build](/docs/configuration/#copr_build) and [test](/docs/configuration/#tests) jobs via `/packit build` and `/packit test` pull request comments,
+  you need to have write access to the repository or be the author of the pull request.
+* Similarly, for retrying the [propose downstream job](/docs/configuration/#propose_downstream) via `/packit propose-downstream` issue comment,
+  you need to have write access to the repository.
+
+
+## 3. Configuration
+
+Packit uses a configuration file to let Packit know what to do, when to do it and how.
```

Comment:
Suggestion: `Packit uses a configuration file for specifying what to do, when and how to do it.`